crates/ruff_python_trivia/src/tokenizer.rs

```diff
@@ -45,49 +45,27 @@ pub fn lines_before(offset: TextSize, code: &str) -> u32 {
 }
 
 /// Counts the empty lines between `offset` and the first non-whitespace character.
+#[allow(clippy::cast_possible_truncation)]
 pub fn lines_after(offset: TextSize, code: &str) -> u32 {
-    let mut cursor = Cursor::new(&code[offset.to_usize()..]);
-
-    let mut newlines = 0u32;
-    while let Some(c) = cursor.bump() {
-        match c {
-            '\n' => {
-                newlines += 1;
-            }
-            '\r' => {
-                cursor.eat_char('\n');
-                newlines += 1;
-            }
-            c if is_python_whitespace(c) => {
-                continue;
-            }
-            _ => {
-                break;
-            }
-        }
-    }
-
-    newlines
+    SimpleTokenizer::starts_at(offset, code)
+        .take_while(|token| {
+            token.kind == SimpleTokenKind::Newline || token.kind == SimpleTokenKind::Whitespace
+        })
+        .filter(|token| token.kind == SimpleTokenKind::Newline)
+        .count() as u32
```

Comment:
It seemed reasonable to me to rewrite this one too for consistency, but don't feel strongly.

Reply:
I prefer the old implementation to keep it consistent with `lines_before`.

Using the string-based approach also has the benefit that we can get away with much simpler code (`lines_after is 20 lines, `SimpleTokenizer` is a couple of hundred lines).